conf/messages

```diff
@@ -77,7 +77,7 @@ estimated_tax_liability.calc-breakdown.reliefs                  = Your tax relie
 
 estimated_tax_liability.calc-breakdown.total-estimate           = Your Income Tax and National Insurance estimate
 estimated_tax_liability.estimated-date                          = Estimate up to your <span id="calc-date">{0}</span> submission
-estimated_tax_liability.no-data                                 = Once you''ve submitted a report using your accounting software, you can view your tax estimate here.
+estimated_tax_liability.no-data                                 = You don''t have an estimate right now. We''ll show your next Income Tax estimate when you submit a report using software.
```

Comment:
@LadonJackson - I don't think this is the right message. It should be the one on the `estimates` page not the `estimated_tax_liability` page. We should probably refactor `estimated_tax_liability` to be `calculation` so that it's clearer.